submissions/shramkoweb/js-dom/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="main.css">
+    <title>Dom - Kottans</title>
+  </head>
+  <body>
+    <main class="main">
+      <aside class="aside">
+        <ul class="frameworks-list"></ul>
+      </aside>
+
+      <section class="content"></section>
```

Comment:
It seems your `section` doesn't have any title. It can cause warnings during the validation of your layout

Reply:
Yes. It's not error. 
In u opinion I need add hidden title ? Or not use semantics correct in this case section ?

Reply:
https://validator.w3.org/nu/?doc=https%3A%2F%2Fshramkoweb.github.io%2Fkottans-dom%2F

Reply:
It's not error. Only warn.
In this case section is more semantics correct.

Reply:
let me add smt as well. 
I also wanted to highlight that semantically there's no sense in <section> if it doesn't have a Header. It's better to add a hidden title. 
Correct, it's not an error, it's a recommendation. But, for example, Vadim Makeev usually says "If you don't want to add a hidden title, let it be <div>, not <section>".
@Shramkoweb 

ref: http://html5doctor.com/avoiding-common-html5-mistakes/
<img width="554" alt="Screenshot 2020-10-16 at 10 44 10" src="https://user-images.githubusercontent.com/25491366/96227336-a86d9200-0f9c-11eb-95dd-7a510636719b.png">

Reply:
@yeromin ok, thx add hidden title =)